taxi/views.py

```diff
@@ -60,17 +62,38 @@ class CarListView(LoginRequiredMixin, generic.ListView):
 
 class CarDetailView(LoginRequiredMixin, generic.DetailView):
     model = Car
+    template_name = "taxi/car_detail.html"
+    context_object_name = "car"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        car = self.get_object()
+        context["driver_ids"] = car.drivers.values_list("id", flat=True)
+
+        return context
+
+    def post(self, request, *args, **kwargs):
+        car = self.get_object()
+        if "assign" in request.POST:
+            print(request.POST, request.user)
```

Comment:
Remove the `print` statement used for debugging purposes. It's not suitable for production code.